CHANGES.rst

```diff
@@ -69,8 +69,8 @@ Improvements
   thanks :user:`SegiNyn`)
 - Use a more compact registration ticket QR code format which is faster to scan and less
   likely to fail in poor lighting conditions (:pr:`6123`)
-- Add a legend to the category calendar, allowing to filter events either by category, venue or room
-  (:issue:`6105, 6106, 6128, 6148, 6149`, :pr:`6110, 6158`, thanks :user:`Moliholy, unconventionaldotdev`)
+- Add a legend to the category calendar, allowing to filter events either by category, venue, room or keywords
```

Comment:
I think this should be wrapped somewhere now (check the surrounding line lengths.

I would also add another (new) changelog entry for the new feature of being able to restrict the available keywords that can be used.

Reply:
Entry added.